packages/labs/src/components/select/select.tsx

```diff
@@ -40,6 +40,13 @@ export interface ISelectProps<T> extends IListItemsProps<T> {
      */
     itemRenderer: (itemProps: ISelectItemRendererProps<T>) => JSX.Element;
 
+    /**
+     * Whether the component is non-interactive.
+     * Note that you'll also need to disable the component's children, if appropriate.
```

Comment:
Ah, this might be enough. Thoughts @llorca?

Reply:
Seems fine to me